airbyte-oauth/src/main/java/io/airbyte/oauth/BaseOAuth2Flow.java

```diff
@@ -75,25 +77,61 @@ public BaseOAuth2Flow(final ConfigRepository configRepository,
   }
 
   @Override
+  @Deprecated
   public String getSourceConsentUrl(final UUID workspaceId, final UUID sourceDefinitionId, final String redirectUrl)
```

Comment:
do we need to keep this method, or should it be removed in this PR? (not sure what depends on it so just asking, if easy to remove here probably better)

Reply:
I removed it and added test to make sure the ones that need to be backward compatible can still be used without `authConfigSpecification`